src/view/frontend/templates/category/remove_category_filter_callback.phtml

```diff
@@ -0,0 +1,9 @@
+<?php /** @var Magento\Framework\View\Element\Template $block */ ?>
+
+<script>
+    document.addEventListener('ffReady', function (ff) {
+        ff.resultDispatcher.addCallback("asn", function (asnData) {
+            asnData.splice(asnData.findIndex(g=>g.__associatedFieldNameV4 === '<?= $block->getData('category_path_field_name')?>'), 1)
+        });
```

Comment:
This is fine but have one flaw, it will not work with the versions other than NG v4
we have to cover also NG v3 an 7.3